wc-calypso-bridge-class.php

```diff
@@ -79,10 +79,16 @@ public function includes() {
 		/** API includes */
 		include_once( dirname( __FILE__ ) . '/api/class-wc-calypso-bridge-currencies-controller.php' );
 		include_once( dirname( __FILE__ ) . '/api/class-wc-calypso-bridge-settings-email-groups-controller.php' );
-		
+
 		if ( class_exists( 'MailChimp_Woocommerce' ) ) {
 			include_once( dirname( __FILE__ ) . '/api/class-wc-calypso-bridge-mailchimp-settings-controller.php' );
 		}
+
+		/**
+		 * Store Stats Testing. woocommerce-analytics will eventually land in Jetpack.
+		 * See https://github.com/Automattic/jetpack/pull/8296 for more details
+		 */
+		include_once( dirname( __FILE__ ) . '/inc/woocommerce-analytics/wp-woocommerce-analytics.php' );
```

Comment:
do we want to add a blog ID check here, or in `isActiveStore` maybe?

Reply:
Lets add it here. I'm copy/pasting the entire folder from the original PR (https://github.com/Automattic/jetpack/pull/8296), so it would be good to keep Calypso Bridge specific things outside of `/inc/woocommerce-analytics`.